.github/workflows/standard.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       # Extras to be installed only for conda-based testing:
       conda-testing-extras: ""
       # Extras to be installed only for pip-based testing:
-      pip-testing-extras: "PyQt5"
+      # Set numpy<2 for now, remove when xraylib updates
+      pip-testing-extras: "PyQt5 numpy<2"
```

Comment:
I thought it might be good to pin this in ci-helpers as well, but if we do that we'll never find all the other incompatibilities.

Reply:
Yeah, I think I'd rather learn about all the failures...